Modules/Scripted/Home/Home.py

```diff
@@ -273,6 +273,8 @@ def setMetadata(self, data):
 class AnnotationManager:
   """Manage serialization and bookkeeping for a collection of annotations."""
 
+  FORMAT_VERSION = '1.0.0'
```

Comment:
Waiting we finalize the update of coordinate et al discussed with Lydia, I suggest to use `2020.09.18`. This will be consistent with [AnnotationFileFormat.md](https://github.com/BICCN/cell-locator/blob/master/Documentation/AnnotationFileFormat.md).

To provide some context:

I currently identified previous version using `YYYY-MM-DD` in the documentation (see [AnnotationFileFormat.md](https://github.com/BICCN/cell-locator/blob/master/Documentation/AnnotationFileFormat.md)) and have been using `YYYYMMDD` in converted script I am finalizing in  https://github.com/BICCN/cell-locator/pull/160 (which we will consolidate with https://github.com/AllenInstitute/cell_locator_tools).

Reply:
Another approach would be to identify the current version as `0.9.0` ... this means for format is not yet stable.

We would also updated the documentation  accordingly to indicate that `2020.09.18` is `0.9.0`, and retro identify the previous version as 0.8.0, 0.7.0.

That way, once  we finalize the coordinate system consolidation, we mark the format as 1.0.0, and start proper semantic versioning

Reply:
My only concern is that if we choose a version like 2020.09.18, then later switch to semantic versioning, then we will have version `2020.x.x` < `1.x.x`. I would prefer to stick with a scheme that would be consistent in the future.

Perhaps we could do something like `0.9.0+2020.09.18` or `1.0.0-alpha+2020.09.18`? My understanding is that the `-alpha` and `+metadata` sections are standard in semantic versioning. That might be a good way to keep things consistent but also include the important information.

Reply:
@jcfr do you have any thoughts on this? Otherwise I would choose `0.9.0+2020.09.18` as the version string.

Reply:
> My only concern is that if we choose a version like 2020.09.18, then later switch to semantic versioning, then we will have version 2020.x.x < 1.x.x which is inconsistent. I would prefer to stick with a scheme that would be consistent in the future.

Makes sense.

Reply:
When the format stabilize and we release `1.0.0`, I suggest we remove the metadata.

```
0.1.0+2020.09.18
0.2.0+2021.MM.DD
0.3.0+2021.MM.DD
...
1.0.0
```